### PR TITLE
Integration of multiple images on user feed cards

### PR DESCRIPTION
**Pull request by Zekereyez**
Context
Before, users were only able to see a user's first profile image. Now, users are allowed to tap on the right or left half of the image view of cards on the card feed and see users images if they have more. 

This diff
Introduces an array to store the users photos, tap gestures, and two new UIViews that are placed on the left and right half of the image view of the card view to handle gesture recognizers and their actions respectively. 

What’s next
Adding user social media links onto profile

TEST PLAN:

Demo: https://imgur.com/a/shVclHe

Thoroughly this feature as seen in demo
- Tapping on the left half means to go back one index of photos
- A user cannot go back one index if currently on the first photo
- Tapping on the right half means to go forward one index
- A user cannot go forward one index if currently on the last photo

### CODE REVIEW

File: Squablr/Views/CardView.m [hunk 1]
```diff
@@ -30,15 +30,31 @@ - (instancetype)initWithBounds:(CGRect)frame profile:(Profile *)profile {
     if (self) {
         [self setup];
         self.profile = profile;
+        self.profilePictureIndex = 0;
+        // Creating invisible half views for the tap gesture recognizer
+        UIView *leftView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 175, 500)];
```
Comment (rhammoud0418) [inline Squablr/Views/CardView.m:35]: Not super important but generally when we're setting frames / constraints like this we either want to use auto layout or do some sort of "manual" auto layout by figuring out what this frame should be based on the frames of the relevant views

File: Squablr/Views/CardView.m [hunk 2]
```diff
@@ -30,15 +30,31 @@ - (instancetype)initWithBounds:(CGRect)frame profile:(Profile *)profile {
...
+        UIView *rightView = [[UIView alloc] initWithFrame:CGRectMake(175, 0, 175, 500)];
+        // Adding tap gesture recognizers for user photos
+        UITapGestureRecognizer *tappedOnLeftHalf = [[UITapGestureRecognizer alloc]
```
Comment (rhammoud0418) [inline Squablr/Views/CardView.m:38]: This init is getting pretty big, I would recommend splitting this up into multiple functions. Like setupSubviews, setupGestureRecognizers, setupStyles, etc.

File: Squablr/Views/CardView.m [hunk 3]
```diff
@@ -120,4 +136,21 @@ - (void)setup {
     self.backgroundColor = [UIColor systemGrayColor];
 }
 
+- (void) nextProfileImage {
```
Comment (rhammoud0418) [inline Squablr/Views/CardView.m:139]: Nice this is a great way of implementing this functionality!

**Review by rhammoud0418 — COMMENTED** [review]
Few recommendations but overall LGTM!

File: Squablr/Views/CardView.m (hunk 2 above)
Comment (Zekereyez) [reply Squablr/Views/CardView.m:38]: Got it

File: Squablr/Views/CardView.m (hunk 1 above)
Comment (Zekereyez) [reply Squablr/Views/CardView.m:35]: I thought when I use initWithFrame it specifies where on the screen the view is? Should it be paired with auto layout regardless?

**Pull request merged**